### PR TITLE
Update connect-github.md

### PR DESCRIPTION
**Pull request by will-sargent-dbtlabs**
## What are you changing in this pull request and why?
Added some additional Language for ST customers wanting to use On-Prem Github via the native integration.

### CODE REVIEW

File: website/docs/docs/cloud/git/connect-github.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Connecting your GitHub account to dbt Cloud provides convenience and another lay
 ## Prerequisites
 
 - For On-Premises GitHub deployment, reference [importing a project by git URL](/docs/cloud/git/import-a-project-by-git-url) to set up your connection instead. Some git features are [limited](/docs/cloud/git/import-a-project-by-git-url#limited-integration) with this setup.
+   - NOTE: "Single Tenant" dbt Customers have additional options available to connect to an On-Premises GitHub deployment using the native integration, making it possible to use all the features of the integration.  Additonal networking configuration prerequisites will apply and our infrastructure team will need to coordinate some information to complete the process. Please contact dbt Labs support or your dbt Cloud account team to discuss the specifics for your tenant.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/git/connect-github.md]: small tweak and adding an example of features

```suggestion
  * **Note** &mdash; [Single tenant](/docs/cloud/about-cloud/tenancy#single-tenant) accounts offer enhanced connection options for integrating with an On-Premises GitHub deployment setup using the native integration.  This integration allows you to use all the features of the integration, such as triggering CI builds. The dbt Labs infrastructure team will coordinate with you to ensure any additional networking configuration requirements are met and completed. To discuss details, contact dbt Labs support or your dbt Cloud account team.
```

**Comment by mirnawong1** [comment]
hey @will-sargent-dbtlabs ! thanks so much for opening this pr up - small tweak and let me know what you think!

**Comment by will-sargent-dbtlabs** [comment]
Looks great, thanks @mirnawong1 !

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
🚢

**Pull request merged**